.github/workflows/linux_packages.yaml

```
@@ -0,0 +1,620 @@
+          echo "✅ ARM64 .deb package verification passed"
+
+      - name: Upload .deb artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: hyprnote-${{ env.VERSION }}-${{ matrix.arch }}.deb
+          path: ${{ steps.find-deb.outputs.deb_path }}
+          if-no-files-found: error
+
+      - name: Upload .deb to release
+        if: github.event_name == 'release'
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.8)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)